app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,18 @@
+class SessionsController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:create]
+
+  def create
+    session.update(
+      current_user: User.build_from(auth_hash),
```

Comment:
We've pondered the security aspect of this and we are still on the fence.

One alternative would be to keep track of logged in users in an hash and have the ability to look them up via uid. At that point, we might as well just move sessions into Redis, for instance.